packages/paste-core/components/avatar/src/utils.ts

```diff
@@ -119,3 +122,47 @@ export const getComputedTokenNames = (
   }
   throw new Error('[Avatar]: size must be a string or an array');
 };
+
+/*
+ * Spacing for Avatar Group
+ */
+
+export const getGroupSpacing = (size: IconSize, variant: AvatarVariants): SpaceOptions => {
+  if (variant === 'user') {
+    switch (size) {
+      case 'sizeIcon10':
+      case 'sizeIcon20':
+      case 'sizeIcon30':
+        return 'spaceNegative10';
+      case 'sizeIcon40':
+      case 'sizeIcon50':
+      case 'sizeIcon60':
+      case 'sizeIcon70':
+        return 'spaceNegative20';
+      case 'sizeIcon80':
+      case 'sizeIcon90':
+        return 'spaceNegative30';
+      case 'sizeIcon100':
+        return 'spaceNegative40';
+      case 'sizeIcon110':
+        return 'spaceNegative50';
+    }
+  } else if (variant === 'entity') {
```

Comment:
suggestion: you might not need this else if you provide a default case with the next switch statement, as it looks like it's just `spaceNegative20` if all else fails